CHANGELOG.md

```diff
@@ -11,6 +11,12 @@ the code was deployed.
 
 ## [Unreleased]
 
+## [13.9.0] - 2024-05-08
```

Comment:
This does not need to be added until we deploy to production. The changes should remain under [Unreleased] and all changes, including work done for other tasks will be moved under the new version when we deploy.

Reply:
Ok! Sorry about that, I removed what I had and then left it as [xx.x.x] for when we deploy.